api/rpc/version/version.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package version;
+
+import "google/api/annotations.proto";
+
+service Version {
+  rpc List (ListRequest) returns (ListReply) {
+    option (google.api.http) = {
+      get: "/v1/version",
```

Comment:
about the .pb.gw.go, please have a look at the Makefile, the PROTOGWTARGETS are not dynamically set, you should add the new files in this list.